example.py

```diff
@@ -6,6 +6,24 @@
 from threads_api.src.http_sessions.requests_session import RequestsSession
 from threads_api.src.http_sessions.aiohttp_session import AioHTTPSession
 
+
+async def example():
```

Comment:
This example doesn't conform to the rest of the examples. The username and password are not brought from env vars, the function name is too generic, and there is no line at the end of the file to toggle it.